game/scripts/npc/abilities/tinker_defense_matrix.txt

```diff
@@ -37,7 +37,7 @@
     {
       "damage_absorb" //OAA
       {
-        "value"                                           "100 200 300 400 800 1200"
+        "value"                                           "110 180 250 320 640 960"
```

Comment:
does level 1 value need to be buffed? prob would do 80 160 240 320 or 95 170 245 320

Reply:
Anything below 100 feels bad at lvl 1. This should be fine.